src/components/profile/ProfileService.js

```diff
@@ -262,11 +298,47 @@ goog.require('ga_urlutils_service');
               .attr('class', 'ga-profile-label ga-profile-label-y')
               .attr('transform', 'rotate(-90)')
               .attr('y', 0 - options.margin.left)
-              .attr('x', 0 - height / 2 - 20)
+              .attr('x', 0 - height / 2 - 30)
               .attr('dy', '1em')
+              .attr('font-size', '0.95em')
               .text($translate.instant(options.yLabel) + ' [m]');
 
-           return element;
+          //Total Elevation Difference
+          group.append('text')
+              .attr('class', 'ga-profile-label ga-profile-elevation-difference')
+              .attr('x', 0)
+              .attr('y', height + options.margin.bottom - 1)
```

Comment:
Ok makes sense.
Since this height is used 3 times create a variable with `height + options.margin.bottom - 1` and add a small comment explaining the reason of using `-1`